VERSION

```diff
@@ -1 +1 @@
-10.13.0b1
+11.0.0
```

Comment:
This version is suppose to point to the latest beta release which is 10.13.0b1

Reply:
ah my bad, will fix

Reply:
this should be fixed